hm10_3.py

```diff
@@ -0,0 +1,56 @@
+""" Task 3: function arguments are positive numbers """
+
+
+def typed(*arg_types):
+
+    """ Function takes arguments _any """
+
+    def decorator(func):
+
+        """ Function takes arguments func """
+
+        def wrapper(*args):
+
+            """ Function takes arguments in any value """
+
+            converted_args = []
+            for arg, arg_type in zip(args, arg_types):
+                converted_arg = arg_type(arg)
+                converted_args.append(converted_arg)
+            return func(*converted_args)
+        return wrapper
+    return decorator
+
+
+@typed(str, int)
+def add(a, b):
+
+    """ Function work with str and int """
+
+    return a + str(b)
+
+
+assert add("9", 9) == "99", "Error processing (str, int)"
+assert add(9, 9) == "99", "Error processing (str, int)"
+assert add("9", "9") == "99", "Error processing (str, int)"
```

Comment:
what if the count of arguments is 3?

Reply:
Chanched arguments for 3 in first variant (not with #), ok.
But I understand, it also could be 5, 100, 100500. And I do not understand, how to fix it. 
If it needs 3 - done
If it needs 5, 100, 100500 - and as for me it should be so - I do not nderstand the correct fix, please help

Reply:
Could you please rephrase your question , I'm not sure that understood it

Reply:
I do not understand is it difference between 3 arguments and indefinite quantity. 
Now I fix it for 3 arguments, but you can ask for 4, 7, 100 arguments.  In that cause, should I add the 4th agrument or my cod is incorrect? If it is incorrect - where is the mistake, I can not understand.

Reply:
In general, your solution will work with different counts of arguments. It uses `*args ` and`for`.
No need to write additional cases